core/src/main/scripts/migrate_db.py

```diff
@@ -0,0 +1,204 @@
+#! /usr/bin/env python
+
+# imports
+import os
+import sys
+import getopt
+import MySQLdb
+
+# Globals
+
+ERROR_FILE = sys.stderr
+OUTPUT_FILE = sys.stdout
+
+DATABASE_HOST = 'db.host'
+DATABASE_NAME = 'db.portal_db_name'
+DATABASE_USER = 'db.user'
+DATABASE_PW = 'db.password'
+VERSION_TABLE = 'info'
+VERSION_FIELD = 'DB_SCHEMA_VERSION'
+
+class PortalProperties(object):
+    """ Properties object class, just has fields for db conn """
+
+    def __init__(self, database_host, database_name, database_user, database_pw):
+        self.database_host = database_host
+        self.database_name = database_name
+        self.database_user = database_user
+        self.database_pw = database_pw
+
+def get_db_cursor(portal_properties):
+    """ Establishes a MySQL connection """
+
+    try:
+        connection = MySQLdb.connect(host=portal_properties.database_host, 
+            port = 3306, 
+            user = portal_properties.database_user,
+            passwd = portal_properties.database_pw,
+            db = portal_properties.database_name)
+    except MySQLdb.Error, msg:
+        print >> ERROR_FILE, msg
+        return None
+
+    if connection is not None:
+        return connection.cursor()
+
+def get_portal_properties(properties_filename):
+    """ Returns a properties object """
+    
+    properties = {}
+    properties_file = open(properties_filename, 'r')
+
+    for line in properties_file:
+        line = line.strip()
+
+        # skip line if its blank or a comment
+        if len(line) == 0 or line.startswith('#'):
+            continue
+        
+        # store name/value
+        property = line.split('=')
```

Comment:
If I used that, we would have to change how our properties files are structured. ConfigParser needs section headers, which aren't currently in our properties files.